src/uint32.ts

```diff
@@ -0,0 +1,267 @@
+// SPDX-License-Identifier: WTFPL
```

Comment:
to be clear, this is a relicensing

uint32.js license (https://github.com/fxa/uint32.js?tab=readme-ov-file#license) is:

> Do, what You want. This library was designed to be a copy/paste template. It would be nice to hear from You, if You used this library or if You just copy/pasted some source. It would be nice, if you added a "contains code of Franz X Antesberger" or something like that or a ref to the github project to your license information or elsewhere.

WTFPL (https://spdx.org/licenses/WTFPL.html) is:

> DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
Version 2, December 2004
>
> Copyright (C) 2004 Sam Hocevar <sam@hocevar.net>
>
> Everyone is permitted to copy and distribute verbatim or modified copies of this license document, and changing it is allowed as long as the name is changed.
>
> DO WHAT THE FUCK YOU WANT TO PUBLIC LICENSE
TERMS AND CONDITIONS FOR COPYING, DISTRIBUTION AND MODIFICATION
>
> 0. You just DO WHAT THE FUCK YOU WANT TO.

some sentiment in common though